XamarinCommunityToolkit/Helpers/SafeArea.shared.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Helpers
+{
+    [TypeConverter(typeof(SafeAreaTypeConverter))]
+    public struct SafeArea
+    {
+        readonly bool isParameterized;
```

Comment:
Should this be `_isParameterized`?

Reply:
> Should this be `_isParameterized`?

No, it shouldn't according to our code style :)